app/src/main/kotlin/Twenty48/Game2048State.kt

```diff
@@ -0,0 +1,25 @@
+package Twenty48
+
+import Twenty48.Game2048
+
+// open class Game2048State(gameObject: Game2048) {
```

Comment:
Clean up all the commented code in this class, if they are not used, remove them.